client/components/credit-card-form-fields/index.jsx

```diff
@@ -90,7 +92,13 @@ export class CreditCardFormFields extends React.Component {
 	};
 
 	shouldRenderEbanxFields() {
-		return shouldRenderAdditionalEbanxFields( this.getFieldValue( 'country' ) );
+		// The add/update card endpoints do not yet process Ebanx payment details
+		// so we only show Ebanx fields at checkout,
+		// i.e., when there is a current transaction.
```

Comment:
Should `i.e.,` be `i.e.` instead?

Reply:
This is a result of the Americanization of my spelling :) Both are acceptable, depending on which side of the Atlantic ocean you live.

Reply:
TIL :)